### DISCUSSION TITLE
Getting information workflow log

### DISCUSSION THREAD

**Discussion by nuno-m-pereira-alb** [community/community#58547] [Actions]
### Select Topic Area

Question

### Body

Good morning, 
I need to input on my code, my log failed about workflow run and I have used GitHub CLI, with following command:  **gh run view** 
However, I can't use only tag --job, because we need to input a runner ID and step either.

What I have tried:  **_gh run view 5246373888 --job 9475146685 --log_**

Message received: 
**_! both run and job IDs specified; ignoring run ID  
failed to get job: HTTP 404: Not Found_**

Anyone have an idea how can I get error workflow without use UI interface GitHub, please ?

**Comment by airtower-luna** [comment]
What exactly are you trying to achieve? `gh run view --log` should give you the log of *either* all jobs in a workflow run, *or* one specific job (which is part of a workflow run). The error message tells you you have to use one of the two options.

**Reply by nuno-m-pereira-alb** [reply]
I'm trying to achieve a workflow error with a ID RUN or ID JOB, but when you write _gh run view ID_RUN --log_ it doesn't return anything

**Reply by airtower-luna** [reply]
I tried just that (replacing `ID_RUN` with one of my workflow run IDs), and I got the full logs. I assume there's some relevant detail that's missing, can you share *exactly* how you're using the command, and the full output? :thinking:

**Reply by nuno-m-pereira-alb** [reply]
I used the following command but it doesn't return any output:

_gh run view 5246373888 --log_

I needed some relevant detail output about these log, like the output workflow, as we watch on UI GitHub when I run a workflow. 

![image](https://github.com/community/community/assets/127102695/c316c3e6-1898-458f-ab2c-2c38a09956c9)

![image](https://github.com/community/community/assets/127102695/86739c69-18c3-43a0-a933-41276b289dce)

**Reply by airtower-luna** [reply]
That is baffling to say the least, for me retrieving the run ID and then log the same way works just fine.

Possibly non-obvious question: Has the run finished?

**Reply by nuno-m-pereira-alb** [reply]
Yes, It finished with error. 
That error I would like to get on code but with this command, I'm not get. 
I remember, do we need some token (OAuth) to access with these workflow data log? 🤔

**Reply by airtower-luna** [reply]
Yes, definitely. Assuming you've done `gh auth login` it should be stored in a `gh` config file, or it might be read from the `GH_TOKEN` or `GITHUB_TOKEN` environment variable. You can check the status with `gh auth status`, that should also tell you the scopes your token has.
